tests/unittest_nodes.py

```diff
@@ -1592,6 +1595,7 @@ def func():
     """
     )
     assert node.doc is None
+    assert node.doc_node is None
```

Comment:
Some additional tests would be appreciated 😄

Reply:
any idea what to test?

Reply:
Sorry I misread this test. This is indeed correct.

Could you add three more tests scenarios? One for `ClassDef` (both with and without docstring), `Module` (with and without) and `NodeNG` that asserts that `doc_node is None`. That way we know we initialise the attribute correctly on all other nodes!

Reply:
So, should I just add test classes to unittest_nodes.py? Or is there a better way/place?

Reply:
Yeah although `ClassDef` and `Module` are likely in `tests/unittest_scoped_nodes.py`. All nodes have a existing test class so you should be able to append to them.

Reply:
I just added the tests for module node. Can you please check if they are fine? I didn't check the line, column etc. since this is already done in test_get_doc

Reply:
Please do add these as well. We never know what intricacies could break the expected behaviour. Especially considering `Module` is such a different compared to `ClassDef` and `FunctionDef`.
Let's make sure we test all expected attributes and values of the `doc_node` attribute. It adds a little bit of extra code to our test suite but makes sure they are much more robust. We could then also add these to `ClassDef`.